src/components/TeamMemberList.js

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import TeamMember from "./TeamMember";
+import axios from "axios";
+
+const TeamMemberList = (props) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const fetchUsers = async () => {
+      const response = await axios.get(process.env.REACT_APP_ENDPOINT_URL);
```

Comment:
and second thing, async/await in yours way is nice, when everything works :) but you need to add come catch block and by the way read about finally also - it's super useful:
example:
https://stackoverflow.com/a/50373478

Reply:
Added try/catch block with custom thrown error.